Project EFT/Data Classes/Submission.cs

```diff
@@ -17,5 +18,15 @@ public Submission(string content, DateTime submissionDate, int userID)
             SubmissionDate = submissionDate;
             UserID = userID;
         }
+
+        //for posterity sake, this used to be an override of Equals, which required a custom GetHashCode function as well...it was never committed, for good reason...
+        //I will allow you to picture what that monstrosity looked like
+        public bool IsEqual(Submission subs)
+        {
+            return (this.Content.Equals(subs.Content)
+                    && this.SubmissionDate.Equals(subs.SubmissionDate)
+                    && this.UserID == subs.UserID);
+        }
+
```

Comment:
should probably be virtual to allow subclasses to override and to avoid subclasses being able to check if they're equal to base / other subclasses besides their own

Reply:
Addressed in above reply.  When this is marked as virtual, if the AnswerSubmissions is overridden, then the input parameter would have to be Submission, not AnswerSubmission, requiring a cast with a try/catch